server/src/test/java/com/defold/extender/IntegrationTest.java

```diff
@@ -110,15 +110,18 @@ public static List<TestConfiguration> data() {
                 // "a" is a made up sdk where we can more easily test build.yml fixes
                 // new DefoldVersion("a", new Version(0, 0, 0), new String[] {"armv7-android", "x86_64-win32"} ),
 
-                // // ??????? https://github.com/defold/defold/releases/tag/1.8.0
-                new DefoldVersion("9141d9d3605e3f5d51c71293116d769da2613d39", new Version(1, 8, 0), new String[] {"armv7-android", "x86_64-linux", "x86_64-win32", "js-web", "wasm-web"}),
-
-                // // 2024-06-25 https://github.com/defold/defold/releases/tag/1.9.0
-                new DefoldVersion("d6882f432beca85d460ec42497888157c356d058", new Version(1, 9, 0), new String[] {"armv7-android", "x86_64-linux", "x86_64-win32", "js-web", "wasm-web"}),
-
+                // // 2024-08-20 https://github.com/defold/defold/releases/tag/1.9.1
+                new DefoldVersion("691478c02875b80e76da65d2f5756394e7a906b1", new Version(1, 9, 1), new String[] {"armv7-android", "x86_64-linux", "x86_64-win32", "js-web", "wasm-web"}),
```

Comment:
Updates sdk version to more recent. Doesn't include 6 last version because it leads to grow overall test execution time.